docs/src/__examples__/Tag/DEFAULT.tsx

```diff
@@ -3,8 +3,13 @@ import { Tag } from "@kiwicom/orbit-components";
 
 export default {
   Example: () => <Tag>Bags</Tag>,
-  info: {
-    title: "Default tag",
-    description: "By default, tags are not selected and cannot be removed.",
-  },
+  exampleKnobs: [
+    {
+      component: "Tag",
+      knobs: [
+        { name: "selected", type: "boolean", defaultValue: false },
+        { name: "size", type: "select", options: ["small", "normal"], defaultValue: "normal" },
+      ],
+    },
+  ],
```

Comment:
The example id in Tag MDX file should be updated to `Tag-default`. I'd stop the build if this happens, not sure what's the proper way to do it.